app/editor/src/features/admin/ingests/configurations/DbMigration.tsx

```diff
@@ -89,6 +92,22 @@ export const DbMigration: React.FC = (props) => {
           is&nbsp;{minMigrationIngestSpanInDays} Day(s)
         </Col>
       </Row>
+      <Row>
+        <Col flex="1 1 1">
+          <FormikRadioGroup
+            label="Migration Type"
+            name="ImportMigrationType"
+            value={!!values.configuration.importMigrationType ? 'Recent' : undefined}
+            onChange={(val) => {
+              isImportStartDateRequired = val.target.value == 'Recent';
```

Comment:
Fixing linting issues.  If you install some plugins with vscode you should see these before the PR.

Reply:
fixed.